x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -909,14 +1023,15 @@ type LogThresholdRecoveredAlert = {
   getId: () => string;
 } & LogThresholdAlert;
 
-const processRecoveredAlerts = ({
+const processRecoveredAlerts = async ({
```

Comment:
Do we have to `await` the call site now?

Reply:
good catch. added `await`.